packages/react/src/components/LightboxMediaViewer/LightboxMediaViewer.js

```diff
@@ -22,79 +22,45 @@ const { prefix } = settings;
 /**
  * LightboxMediaViewer Component
  *
- * @param {object} props props object
- * @param {string} props.title LightboxMediaViewer media title
- * @param {string} props.copy LightboxMediaViewer media short description
- * @param {string} props.image LightboxMediaViewer responsive image object
- * @param {boolean} props.open sets whether the modal is open/close
- * @param {Function} props.onClose do something on close in addition, return false to completely replace
+ * @param {object} modalProps props object
+ * @param {string} modalProps.title LightboxMediaViewer media title
+ * @param {string} modalProps.copy LightboxMediaViewer media short description
+ * @param {string} modalProps.image LightboxMediaViewer responsive image object
+ * @param {boolean} modalProps.open sets whether the modal is open/close
  * @returns {*} JSX Object
  */
-const LightboxMediaViewer = ({ title, copy, image, open, onClose }) => {
-  /**
-   * Set default image
-   *
-   * @param {*} image responsive image object
-   * @returns {string} default image url
-   */
-  const defaultImage = image => {
-    return image.uri.md
-      ? image.uri.md
-      : image.uri.lg
-      ? image.uri.lg
-      : image.uri.sm;
-  };
-
-  if (!image.uri.lg) {
-    return null;
-  }
+const LightboxMediaViewer = modalProps => {
   return featureFlag(
     DDS_LIGHTBOX_MEDIA_VIEWER,
     <section
       data-autoid={`${stablePrefix}--lightbox-media-viewer`}
       className={`${prefix}--lightbox-media-viewer`}>
-      <ExpressiveModal open={open} fullwidth={true} onClose={onClose}>
+      <ExpressiveModal fullwidth={true} {...modalProps}>
         <ModalBody>
           <div className={`${prefix}--lightbox-media-viewer__container`}>
             <div className={`${prefix}--lightbox-media-viewer__row`}>
               <div
                 data-autoid={`${stablePrefix}--lightbox-media-viewer__image`}
                 className={`${prefix}--lightbox-media-viewer__image`}>
-                <picture>
-                  {image.uri.lg && (
-                    <source
-                      media="(min-width: 672px)"
-                      srcSet={`${image.uri.lg}`}
-                    />
-                  )}
-                  {image.uri.md && (
-                    <source
-                      media="(min-width: 320px)"
-                      srcSet={`${image.uri.md}`}
-                    />
-                  )}
-                  {image.uri.sm && (
-                    <source
-                      media="(max-width: 320px)"
-                      srcSet={`${image.uri.sm}`}
-                    />
-                  )}
-                  <img src={defaultImage(image)} alt={image.alt}></img>
-                </picture>
+                {modalProps.images.map((image, index) => {
+                  return (
+                    <img key={index} src={image.src} alt={image.alt}></img>
```

Comment:
Can we replace with the Image component?

```jsx
<Image {...image} />
```

I believe the `Image` component handles the looping and even ordering now too.

Reply:
Okay @photodow will update accordingly